5_recognition.ipynb

```diff
@@ -238,25 +425,50 @@
     "HSVは、色相(Hue)、彩度(Saturation)、明度(Value)によって色を表現します。今回は色相情報のみを利用してレゴブロックの水色を抽出します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "The color of the Lego block is \"light blue\", so let's use the color to detect the Lego block.\n",
+    "\n",
+    "Color data is stored in RGB format in \"image_data\".\n",
+    "We can use the RGB data as is, but it is known that converting the color space from RGB to HSV makes it more robust against lighting changes.\n",
+    "\n",
+    "HSV expresses color by Hue, Saturation and Value. This time, we will extract the light blue of the Lego block using the hue channel:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# 画像の色相の値を取得します\n",
+    "# 画像の色相の値を取得します Get hue channel\n",
     "h_image = rgbd.get_h_image()\n",
-    "# 色相画像を表示してみます\n",
+    "# 色相画像を表示してみます Display the hue channel\n",
     "plt.imshow(h_image)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "色相画像に対して適当な閾値を設定して、レゴブロックのみが抽出されるようにします。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Set an appropriate threshold for the hue image so that only Lego blocks are extracted:"
```

Comment:
Again, here "Lego blocks" is written, giving the impression that there are multiple Lego blocks that are all the same color.
Am I understanding this correctly?
Or is it the case that there is a single Lego block?

Reply:
This case I think "the Lego block" is appropriate.

We have multiple Lego blocks but the one we spawned is "light blue".

We want to extract that specific Lego block from the input image.

Reply:
Ah...  so perhaps it would be better to say the following since it directly refers to "the blue Lego block":

    "Set an appropriate threshold for the hue image so that only the blue Lego block is extracted:"